### PR TITLE
feat!(backend): Add databse, Untis API and own api routes

### PR DESCRIPTION
**Pull request by ixhbinphoenix**
Man this shouldn't be all in one PR wtf did we do

### CODE REVIEW

File: backend/src/api_wrapper/untis_client.rs
Comment (Sxyntheon) [inline backend/src/api_wrapper/untis_client.rs:1]: holy ass bro how do you write these functions?!

**Review by Sxyntheon — APPROVED** [review]
What you guys did looks pretty good.

File: backend/src/api_wrapper/untis_client.rs
Comment (renkertm) [reply backend/src/api_wrapper/untis_client.rs:1]: I dont know man...

Comment (renkertm) [reply backend/src/api_wrapper/untis_client.rs:1]: My brain is fucked up sometimes

**Review by renkertm — CHANGES_REQUESTED** [review]
Come on first prototype this month

**Comment by renkertm** [comment]
tf i didnt meant to request changes

**Review by renkertm — APPROVED** [review]
wasnt approve the last one? or am i just tired?

**Pull request merged**